test/controllers/apps/letsencrypt_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class Apps::LetsencryptControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Let's add a couple of tests for this :-)

Reply:
Yess

Reply:
Added muchos tests!